### PR TITLE
Added useBaterryStatus Hook

### PR DESCRIPTION
**Pull request by chiragbairwa**
Issue :- [FEAT] Create useBatteryStatus() hook #18

Added **useBatteryStatus** hook ✅
<ul>
<li>The useBatteryStatus() hook should provide information about the battery level and charging status. ✅</li>
<li>Ensure the hook updates the battery status in real-time as it changes. ✅</li>
</ul>

### CODE REVIEW

File: lib/hooks/useBatteryStatus.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react'
+
+export function useBatteryStatus() {
+    const [batteryStatus, setBatteryStatus] = useState({
+        level : 0,
+        status : false
```
Comment (DavidHDev) [inline lib/hooks/useBatteryStatus.js:6]: I would rename this to "isCharging" instead

File: lib/hooks/useBatteryStatus.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    })
+
+    useEffect(()=>{
+      window.onload = () => {
+          if(!navigator.getBattery){
+              return batteryStatus
+          }
+      }
+
+      navigator.getBattery().then((battery) => {
```
Comment (DavidHDev) [inline lib/hooks/useBatteryStatus.js:16]: You should move the check here if the battery status feature is available in the used navigator:
if ('getBattery' in navigator) -> do the stuff

File: lib/hooks/useBatteryStatus.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react'
+
+export function useBatteryStatus() {
+    const [batteryStatus, setBatteryStatus] = useState({
+        level : 0,
+        status : false
+    })
+
+    useEffect(()=>{
+      window.onload = () => {
```
Comment (DavidHDev) [inline lib/hooks/useBatteryStatus.js:10]: it's messy to use window.onload inside a hook, see other comment if you want to check for battery availability

**Comment by DavidHDev** [comment]
One more thing, you did not respect the branch naming convention mentioned in CONTRIBUTING.MD, please do it next time.

**Comment by chiragbairwa** [comment]
I have open a new pull request with all the changes there. https://github.com/DavidHDev/react-haiku/pull/31
Hope it gets approved.

**Pull request closed without merging**